### PR TITLE
Fix documentation of deployVmConsoleProxy feature gate

### PR DESCRIPTION
**Pull request by akrejcir**
**What this PR does / why we need it**:
Setting the feature gate to `false` or nil, will remove the `vm-console-proxy`.

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly

**Release note**:
```release-note
Fix documentation for feature gate "deployVmConsoleProxy".
```

### CODE REVIEW

File: docs/cluster-configuration.md [hunk 1]
```diff
@@ -161,11 +161,8 @@ reverted back to false.
 **Default**: `false`
 
 ### deployVmConsoleProxy Feature Gate
-Set the `deployVmConsoleProxy` feature gate to true to allow SSP operator to deploy its resources. SSP operator will 
-deploy a proxy that provides an access to the VNC console of a KubeVirt Virtual Machine (VM).
-
-**Note**: Once `deployVmConsoleProxy` is set to true, SSP operator will not delete deployed resources if `deployVmConsoleProxy` is 
-reverted back to false.
+Set the `deployVmConsoleProxy` feature gate to `true` to allow SSP operator to deploy its resources. SSP operator will 
+deploy a service that provides new API to generate a time limited token to access the VNC console of a KubeVirt Virtual Machine (VM).
```
Comment (geetikakay) [inline docs/cluster-configuration.md:165]: This looks good. could you please add verification steps to be double sure that setting false/true works as expected.

Comment (akrejcir) [reply docs/cluster-configuration.md:165]: How should these verification steps look like?

Comment (geetikakay) [reply docs/cluster-configuration.md:165]: https://docs.openshift.com/container-platform/4.13/virt/vm_templates/virt-automatic-bootsource-updates.html --> something like this but possibly in downstream.

Comment (codingben) [inline docs/cluster-configuration.md:165]: Why `Setting the feature gate to false or nil, will remove the vm-console-proxy.` wasn't mentioned as well?

**Pull request closed without merging**